runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -377,6 +390,89 @@ else if (containsBoulder) // For traps like deadfalls. This is different because
 		lastTickLocalPlayerLocation = client.getLocalPlayer().getWorldLocation();
 	}
 
+	@Subscribe
+	public void onInteractingChanged(InteractingChanged interactingChanged)
+	{
+		Actor target = interactingChanged.getTarget();
+		Actor source = interactingChanged.getSource();
+
+		mapActorToChinTarget(target, source);
+	}
+
+	/**
+	 *	 				DEAD TRAP MECHANICS
```

Comment:
This section is very helpful to describe what is happening. It is however missing an explicit description that when trap objects (the visual boxes on the floor) spawn, an invisible null npc is spawned at the same location which performs these interactions.